### ISSUE TITLE
Tried on both current Chrome and Firefox, not giving me any buttons next to like button

### ISSUE THREAD

**Issue by RocketDerp** [Tandashi/YTSync-Plugin#48]
### What happened?

Has YouTube layout change broken this extension?

### Where did it happened?

https://www.youtube.com/watch?v=QQD-OvbCfCU

### What browsers are you seeing the problem on?

Firefox, Chrome

### What version are you seeing the problem on?

1.9.1

**Comment by Tandashi** [comment]
Yeah, this Extension has been broken for a while now. YouTube changes so frequently that keeping / maintaining this Extension has become impossible for me, so I stopped developing / fixing it about a year ago.
